pkg/volume/cinder/attacher.go

```diff
@@ -75,9 +78,9 @@ func (attacher *cinderDiskAttacher) Attach(spec *volume.Spec, nodeName types.Nod
 
 	pending, volumeStatus, err := attacher.cinderProvider.OperationPending(volumeID)
```

Comment:
Why does OperationPending return error if it is pending?  Shouldn't err == nil and pending == true?